src/components/App.js

```diff
@@ -1,25 +1,128 @@
-import Header from '../containers/Header';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import {
+  Brand,
+  Page,
+  PageHeader,
+  PageSection,
+  Toolbar,
+  ToolbarGroup,
+  ToolbarItem,
+  DropdownToggle,
+  Dropdown,
+  DropdownItem
+} from '@patternfly/react-core';
 import Overview from '../containers/Overview';
 import Client from '../containers/Client';
 import ErrorMessages from '../containers/ErrorMessages';
-
+import { fetchUserInfo } from '../actions/users';
 import './App.css';
 
-const App = () => (
-  <Router>
-    <div className="App">
-      <ErrorMessages />
-      <Header />
-      <Switch>
-        <Route exact path="/overview" component={Overview} />
-        <Route exact path="/mobileclient/:id" component={Client} />
-        {/* Default redirect */}
-        <Redirect to="/overview" />
-      </Switch>
-    </div>
-  </Router>
-);
-
-export default App;
+class App extends React.Component {
+  state = {
+    isDropdownOpen: false,
+    isIconOpen: false
+  };
+
+  componentWillMount() {
+    this.props.fetchUserInfo();
+  }
+
+  onDropdownToggle = () => {
+    this.setState({ isDropdownOpen: !this.state.isDropdownOpen });
+  };
+
+  onIconToggle = () => {
+    this.setState({ isIconOpen: !this.state.isIconOpen });
+  };
+
+  render() {
+    const userDropdownItems = [
+      <DropdownItem>
+        <a href="/oauth/sign_in">Logout</a>
+      </DropdownItem>
+    ];
+
+    const questionIconItems = [
+      <DropdownItem>
+        <a href="https://docs.aerogear.org/aerogear/latest/getting-started.html">Documentation</a>
+      </DropdownItem>,
+      <DropdownItem>
+        <a href="/">About</a>
+      </DropdownItem>
+    ];
+
+    const PageToolbar = (
+      <Toolbar>
+        <ToolbarGroup>
+          <ToolbarItem>
+            <Dropdown
+              isPlain
+              position="right"
+              isOpen={this.state.isIconOpen}
+              onSelect={this.onDropdownSelect}
+              toggle={
+                <DropdownToggle onToggle={this.onIconToggle} iconComponent={null}>
+                  <OutlinedQuestionCircleIcon />
+                </DropdownToggle>
+              }
+              dropdownItems={questionIconItems}
+            />
+            <Dropdown
+              isPlain
+              position="right"
+              isOpen={this.state.isDropdownOpen}
+              onSelect={this.onDropdownSelect}
+              toggle={
+                <DropdownToggle onToggle={this.onDropdownToggle}>
+                  {this.props.user ? this.props.user.name : 'Unknown'}
+                </DropdownToggle>
+              }
+              dropdownItems={userDropdownItems}
+            />
+          </ToolbarItem>
+        </ToolbarGroup>
+      </Toolbar>
+    );
+
+    return (
+      <Router>
+        <ErrorMessages />
+        <Page
+          header={
+            <PageHeader
+              logo={<Brand src="/img/logo.svg" alt="Mobile Developer Console Logo" />}
```

Comment:
logo={<Brand src="/img/logo.svg" alt="Mobile Developer Console Logo" style={{ width: '300px' }} />}

Reply:
Awesome, thank you! :)